src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { TabProps } from '../../utils/types';
+
+export const Tab: React.FC<TabProps> = ({
+  isActive,
+  title,
+  id,
+  setActiveTabId,
+}) => {
+  return (
+    <li className={isActive ? 'is-active' : ''} data-cy="Tab">
+      <button
```

Comment:
You don't need the `button` here. An anchor inside a button is an error.
Just use `a` tag inside `li` without `button` usage.